packages/client/tests/functional/metrics/tests.ts

```diff
@@ -28,16 +28,16 @@ testMatrix.setupTestSuite(({ provider }) => {
     test('returns metrics in prometheus format', async () => {
       const metrics = await prisma.$metrics.prometheus()
 
-      expect(metrics).toContain('query_total_operations')
-      expect(metrics).toContain('query_total_queries')
-      expect(metrics).toContain('query_active_transactions')
-      expect(metrics).toContain('query_total_elapsed_time_ms_bucket')
+      expect(metrics).toContain('prisma_client_queries_total')
+      expect(metrics).toContain('prisma_datasource_queries_total')
+      expect(metrics).toContain('prisma_client_queries_active')
+      expect(metrics).toContain('prisma_client_queries_duration_histogram_ms_bucket')
 
       if (provider !== 'mongodb') {
-        expect(metrics).toContain('pool_wait_duration_ms_bucket')
-        expect(metrics).toContain('pool_active_connections')
-        expect(metrics).toContain('pool_idle_connections')
-        expect(metrics).toContain('pool_wait_count')
+        expect(metrics).toContain('prisma_client_queries_wait_histogram_ms_bucket')
+        expect(metrics).toContain('prisma_pool_connections_open')
+        expect(metrics).toContain('prisma_pool_connections_idle')
+        expect(metrics).toContain('prisma_client_queries_wait')
```

Comment:
I'll let someone from Client Team approve the PR since I have little context on this.

From my point of view, I think we could test all the metrics available here, for example `prisma_pool_connections_busy` seems to b missing.
Found in
https://github.com/prisma/prisma-engines/pull/3090/files#diff-c9529da676ae473a03f3796f2d150b361e2316734519f084933e6e7fe9ceca43R48-R59

Reply:
That one is missing because it is quite tricky to get it during test setup. DB query has to be in progress at the time we request metric for it to show up. The rest of the metrics are there.